sidebars.js

```diff
@@ -236,7 +236,7 @@ module.exports = {
           type: "category",
           label: "Tutorials",
           collapsed: false,
-          items: ["php/hello-world"],
+          items: ["php/hello-world", "php/subscription-tutorial"],
```

Comment:
prolly add this to the learning paths sidebar section under a new category "Subscription"

Reply:
i cut out the Tutorials section just to be in line with your goals and moved the mention to the intro for now so that it is not an orphan page.